packages/@okta/vuepress-site/docs/concepts/iam-overview/index.md

```diff
@@ -0,0 +1,177 @@
+
+- **High availability:** Ensure that all elements of your IAM solution are fully operational whenever your apps are expected to be operable.
+
+- **High reliability** (highly redundant): Ensure that all elements of your IAM solution are highly reliable and dependable, and that every element of your solution works correctly, every time.
+
+#### Security
+
+- **User storage and password management:** Store and manage information about your users, including their passwords, for authenticating and authorizing users.
+
+- **MFA (Multi-Factor Authentication):** Require a user to verify their identity in different ways. Some of these ways including knowledge, such as answers to questions, biometrics, such as a fingerprint, or possession, such as a key-card. MFA is key to controlling access to your applications.
+
+- **DDoS protection:** Prevent Distributed Denial of Service (DDoS) attacks from blocking legitimate use of your applications.
+
+- **Compliance:** Ensure that your solution complies with specific regulatory requirements for privacy, as well as with industry and local cybersecurity standards. For example, applications that handle healthcare patient data must conform to HIPPA, financial solutions may require SOX or PCI, and any that server Europe must follow GDPR.
+
+- **Data access control:** Develop facilities to grant or deny specific user requests to access apps and resources based on policies, user authentication and authorizations, and other data. Application developers embed access control checks throughout their code to enforce your access requirements.
+
+#### User experience
+
+- **Self service:** Allow end users, especially customers, to self-administer their accounts, including creation, password reset, and originating access requests. This can increase customer satisfaction, reduce friction, and reduce your admin workload.
+
+- **Social auth (social authentication):** Allow end users to sign in with their social media credentials, such as a Facebook or LinkedIn ID, rather than special credentials for your IAM solution. This can enhance customer satisfaction, provide reliable user demographic information, and reduce admin workload. (Social authentication is a special case of *external IdP authentication*, below.)
+
+- **External IdP (external Identity Provider):** Allow users to sign in using a sign-in ID from an external Identity Provider, such as Active Directory, rather than special credentials for your apps. This can increase user satisfaction, and reduce your admin workload.
+
+- **SSO (Single Sign-On):** Allow users to sign in with a single ID to access multiple related applications. This can increase user satisfaction and reduce admin workload. Closely related is FIM *(Federated Identity Management)*, in which users sign log in and access multiple federated external identity providers. It has the same benefits as SSO, and allows users to sign in with their existing sign-in IDs.
```

Comment:
```suggestion
- **SSO (Single Sign-On):** Allow users to sign in with a single ID to access multiple related applications. This can increase user satisfaction and reduce admin workload. Closely related is FIM *(federated identity management)*, in which users sign in and access multiple federated external Identity Providers. It has the same benefits as SSO, and allows users to sign in with their existing sign-in IDs.
```